src/app/component/comments/components/add-comment/add-comment.component.html

```diff
@@ -34,9 +34,16 @@
     </ng-template>
   </form>
 </div>
-<app-comment-body [elements]="elements"
-                  [type]="dataSet.type"
-                  *ngIf="(this.dataSet.type === 'reply' && this.repliesVisibility)
+<div class="comments-loader"
+     *ngIf="elements.length < 1 && elementsArePresent; else comments"
+>
+  <mat-spinner></mat-spinner>
+</div>
```

Comment:
Why did you use this unnecessary wrapping `div`? You can use `*ngIf` and `class` to the component `mat-spinner` directly!

Reply:
Fixed